public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -62,17 +62,19 @@ import {
 } from './useContactPoints';
 import { ContactPointWithMetadata, getReceiverDescription, isProvisioned, ReceiverConfigWithMetadata } from './utils';
 
-enum ActiveTab {
-  ContactPoints,
-  NotificationTemplates,
+export enum ActiveTab {
+  ContactPoints = 'contact_points',
+  NotificationTemplates = 'templates',
 }
 
 const DEFAULT_PAGE_SIZE = 10;
 
 const ContactPoints = () => {
   const { selectedAlertmanager } = useAlertmanager();
-  // TODO hook up to query params
-  const [activeTab, setActiveTab] = useState<ActiveTab>(ActiveTab.ContactPoints);
+  const [queryParams, setQueryParams] = useQueryParams();
```

Comment:
`useURLSearchParams` gives you a native `URLSearchParam` interface to work with. I think it could simplify the code if used